weedcoco/schema/Category.yaml

```
@@ -6,7 +6,7 @@ allOf:
     properties:
       name:
         type: string
-        pattern: "^((crop|weed): ([a-z][a-z ]+|UNSPECIFIED)|crop|weed|none)$"
+        pattern: "^((crop|weed|other): ([a-z][a-z ]+|UNSPECIFIED)|crop|weed|none)$"
```

"other" doesn't need a species, so I guess it should be
```suggestion
        pattern: "^((crop|weed): ([a-z][a-z ]+|UNSPECIFIED)|crop|weed|other|none)$"
```

but then "other" needs a description below.

Maybe "other" is stupid and we should leave it out. At one point I thought we'd use it to represent other objects like a boot or a piece of machinery captured in the image. But maybe it's not useful. As they say, YAGNI.